src/api/user.js

```diff
@@ -1,8 +1,12 @@
 import httpClient from '../services/httpClient';
-import {USER} from '../endpoints';
+import {ME, USERS} from '../endpoints';
```

Comment:
ME - this name a little bit awkward in this context. Could you rename it on CURRENT_USER